### DISCUSSION TITLE
How can I create a folder in a git repository - using the web editor

### DISCUSSION THREAD

**Discussion by lakruzz** [kea-dev/technology-intro#7] [Q&A] [answered]
It seems like the only option I get when I use the web editor on GitHub _(Code >> Add file >> Create new file)_ is to create a _new file_ - but how do I go about it if I want to create the file in a _new folder_?

**Comment by tohasim** [comment, marked as answer]
If you want to create a new file in a new subfolder, you can add _foldername/_ in front of the filename, this will create a subfolder with the name _foldername_ and put the file in there :)
